libs/agent-sm/agent/src/main/java/org/opensearch/javaagent/Agent.java

```diff
@@ -77,7 +77,13 @@ private static AgentBuilder createAgentBuilder(Instrumentation inst) throws Exce
             .with(AgentBuilder.TypeStrategy.Default.REDEFINE)
             .ignore(ElementMatchers.none())
             .type(systemType)
-            .transform(transformer);
+            .transform(transformer)
+            .type(ElementMatchers.named("java.lang.System"))
```

Comment:
Is there any advantage to ElementMatchers.named vs `ElementMatchers.is(System.class)`?

Reply:
I think I just used the old proven "named" but `ElementMatchers.is(System.class)` is definitely cleaner here, since we know the class in advance, thanls!